source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -0,0 +1,73 @@
+.. _kafka-intro:
+
+======================================
+Relational Migrator with Kafka Connect
+======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use Relational Migrator to migrate data to and from Kafka. 
+Relational Migrator can be used as a Kafka Connect plug-in that improves 
+resilience and scalability of big data migration jobs.
+
+Use Cases
+---------
+
+Relational Migrator with Kafka is intended for large and long running jobs. 
```

Comment:
(content updates)
Please be sure this section reflects Tom's comments in the outline doc. 
https://docs.google.com/document/d/18XMcDQflwf7zdfjpjhMZYaBrvWXTsvaU0FuNZfU8yYo/edit#heading=h.wtgv99w5dsen

Reply:
Updated.